src/base/init/install.c

```diff
@@ -237,12 +253,23 @@ static void call_installdos(void)
 		free(boot_dir_path);
 		return;
 	}
-	/* symlink command.com in case someone hits Shift or F5 */
-	ret = asprintf(&system_str,
-			"ln -s ../drives/d/command.com "
-			"../drives/d/kernel.sys "
-			"\"%s\"",
-			boot_dir_path);
+	switch (dos_status) {
+		case 0:
+			/* symlink command.com in case someone hits Shift or F5 */
+			ret = asprintf(&system_str,
+					"ln -s ../drives/d/command.com "
+					"../drives/d/kernel.sys "
+					"\"%s\"",
+					boot_dir_path);
+			break;
+		case 100:
+			ret = asprintf(&system_str,
+					"ln -s ../drives/d/COMMAND.COM "
```

Comment:
It depends on the situation whether they are interchangeable, but  7zip and mcopy can both extract disk images.